styles/globals.css

```diff
@@ -1,16 +1,32 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat+Alternates:wght@900&family=Paytone+One&family=Quicksand&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+
 html,
 body {
   padding: 0;
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
-    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, "Poppins",
+    sans-serif, Quicksand, Paytone One;
 }
 
 a {
   color: inherit;
   text-decoration: none;
 }
 
+:root {
```

Comment:
is root working on this place after html etc.?

Reply:
Yes :)

Reply:
It was there as default setting.